plugin/latexlivepreview.vim

```diff
@@ -268,22 +268,26 @@ EEOOFF
     endif
 
     function! s:ValidateExecutables( context, executables )
+        let l:explicit_engine = get(g:, a:context, '')
```

Comment:
Can we use a different name, because this is used for both engine and previewer?

```suggestion
        let l:user_set = get(g:, a:context, '')
```

Reply:
Done